### PR TITLE
doc(textarea): add textarea component

### PR DESCRIPTION
**Pull request by VMBindraban**

### CODE REVIEW

File: doc/component/form-textarea.md [hunk 1]
```diff
@@ -1,2 +1,37 @@
...
+    name.bind="message" 
+    classes.bind="form-control" 
+    value="Hello!" 
+    placeholder="A message" 
+    autofocus.bind="true" 
+    required.bind="true" 
+    disabled.bind="false" 
+    maxlength.bind="500" 
+    readonly.bind="false" 
+    wrap.bind="hard">
```
Comment (RWOverdijk) [inline doc/component/form-textarea.md]: You can't bind a string. Use regular attribute or supply a view model in the example. (I recommend using regular attributes)

File: doc/component/form-textarea.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
...
+| disabled | boolean | `false` | Specifies that the textarea should be disabled |
+| maxlength | number | | The maximum number of characters allowed in the text area |
+| readonly | boolean | `false` | Specifies that the textarea should be read-only |
+| wrap | string | `'hard'` | Specifies how the text in a text area is to be wrapped when submitted in a form |
+
+## Example
+```html
+<aurelia-textarea 
+    name.bind="message" 
+    classes.bind="form-control"
```
Comment (RWOverdijk) [inline doc/component/form-textarea.md]: No need, this is a default class. Use something else for the example.

File: doc/component/form-textarea.md [hunk 3]
```diff
@@ -1,2 +1,37 @@
...
+| rows | number | | The visible number of lines in the textarea |
+| disabled | boolean | `false` | Specifies that the textarea should be disabled |
+| maxlength | number | | The maximum number of characters allowed in the text area |
+| readonly | boolean | `false` | Specifies that the textarea should be read-only |
+| wrap | string | `'hard'` | Specifies how the text in a text area is to be wrapped when submitted in a form |
+
+## Example
+```html
+<aurelia-textarea 
+    name.bind="message"
```
Comment (RWOverdijk) [inline doc/component/form-textarea.md]: Same here, and other attributes. message is not a bindable. Use regular attribute.

**Review by RWOverdijk — CHANGES_REQUESTED** [review]

File: doc/component/form-textarea.md [hunk 4]
```diff
@@ -1,2 +1,37 @@
...
+| cols | number | `undefined` | The visible width of the textarea |
+| rows | number | `undefined` | The visible number of lines in the textarea |
+| disabled | boolean | `false` | Specifies that the textarea should be disabled |
+| maxlength | number | `undefined` | The maximum number of characters allowed in the text area |
+| readonly | boolean | `false` | Specifies that the textarea should be read-only |
+| wrap | string | `'hard'` | Specifies how the text in a text area is to be wrapped when submitted in a form |
+
+## Example
+```html
+<aurelia-textarea
```
Comment (jeremyvergnas) [inline doc/component/form-textarea.md]: Should be `<form-textarea>`

**Review by RWOverdijk — APPROVED** [review]

**Pull request merged**